examples/01_plotting/plot_transparency.py

```diff
@@ -0,0 +1,295 @@
+r"""
+Plotting images with transparent thresholding
+=============================================
+
+Standard thresholding means that any data below a threshold value
+is completely hidden from view.
+However, "transparent thresholding" allows
+for the same suprathreshold results to be observed,
+while also showing subthreshold information
+with an opacity that fades with magnitude
+(:footcite:t:`Allen2012`, :footcite:t:`Chen2022`,
+:footcite:t:`Taylor2023`, :footcite:t:`Sundermann2024`).
+
+This makes use of the "alpha" value that overlays can have
+when using some plotting functions (like :func:`matplotlib.pyplot.imshow`).
+This "alpha" value goes from 0 (perfectly transparent) to 1 (perfectly opaque).
+
+Consider having an underlay color :math:`RGB_{ulay}`
+and overlay color :math:`RGB_{olay}`,
+where a threshold value T is applied
+and we consider how an element (voxel, node...) with value M is shown.
+
+"opaque" thresholding
+---------------------
+
+if :math:`\lvert M \lvert >= T : alpha=1`,
+meaning the overlay is shown as: :math:`RGB_{olay}`
+else : :math:`alpha=0`, meaning the underlay is shown as: :math:`RGB_{ulay}`
+
+"transparent" thresholding
+--------------------------
+
+The steepness of fading can be linear or quadratic. Linear is shown below.
+
+If :math:`\lvert M \lvert >= T : alpha=1`,
+meaning the overlay is shown as: :math:`RGB_{olay}`.
+
+Otherwise :math:`alpha = (\lvert M \lvert /  T)`,
+merging :math:`RGB_{olay}` and :math:`RGB_{ulay}` as:
+
+:math:`RGB_{final} = (1-alpha) * RGB_{ulay} + alpha * RGB_{olay}`.
+
+In the end, this is just a small tweak for the case of subthreshold data.
+In that case, alpha is nonzero (rather than simply 0).
+
+Additionally, a contour can be placed around the suprathreshold regions,
+to further highlight them.
+
+So, the differences between standard, opaque thresholding
+and the suggested transparent thresholding
+is shown in the rest of this example.
+
+..  admonition:: Benefits
+    :class: important
+
+    Implementing transparent thresholding enables
+    more informative results reporting,
+    enables more accurate interpretations,
+    reduces biases in results and hypersensitive
```

Comment:
It is unclear to me whether it **reduces** "hypersensitivity to non-physiological differences" or **is** "hypersensitive to non-physiological differences"? For either case, I think a little rephrasing would help.

Reply:
Also I don't know how the rest of us feel about it, but personally, I would rephrase this block to read more unbiased. For example, saying like:

"Implementing transparent thresholding **may** enable...." (and then cite the paper)

Reply:
toned things down a bit